src/pages/PaymentSuccess/PaymentSuccess.tsx

```diff
@@ -1,15 +1,24 @@
 import React from "react";
-import { Button, Container } from "react-bootstrap";
+import { Button, Container, Card } from "react-bootstrap";
 import { Link } from 'react-router-dom';
+import Search from "../../components/Search/Search";
+import './PaymentSuccess.css';
 
 function Success() {
     return (
         <div className="payment-success">
             <Container>
-                <h1 style={{color: "limegreen"}}>Your payment was handled successfully!</h1>
-                <p>You will get your features within the next few minutes.</p>
-                <p>If any problems should occur please contact us via <Link to="/feedback">the contact page</Link>.</p>
-                <Link to="/"><Button>Return to main page</Button></Link>
+                <Search />
+                <Card>
+                    <Card.Header>
+                        <Card.Title style={{ color: "#40ff00" }}>Your payment was handled successfully!</Card.Title>
+                    </Card.Header>
+                    <Card.Body>
+                        <p>You will get your features within the next few minutes.</p>
+                        <p>If any problems should occur please contact us via <Link to="/feedback">the contact page</Link>.</p>
```

Comment:
Should the `/feedback` site be renamed to `/contact` ?

Reply:
If so, remember to redirect `/feedback`

Reply:
if so, not in this ticket